map_view/display-drawing-status/src/main/java/com/esri/samples/display_drawing_status/DisplayDrawingStatusSample.java

```diff
@@ -55,12 +56,16 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
+      // authentication with an API key or named user is required to access basemaps and other location services
+      String yourAPIKey = System.getProperty("apiKey");
+      ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
       // create progress bar
       ProgressBar progressBar = new ProgressBar();
       progressBar.setMaxWidth(240.0);
 
-      // create a ArcGISMap with topographic basemap
-      final ArcGISMap map = new ArcGISMap(Basemap.createTopographic());
+      // create an ArcGISMap with a topographic basemap style
```

Comment:
Let's do plain english here as well:
```suggestion
      // create a map with the topographic basemap style
```